package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "babel-install": "^2.1.0",
+    "eslint-config-airbnb-base": "^15.0.0",
```

Comment:
- [ ] The app is throwing this error, kindly fix it 
![image](https://user-images.githubusercontent.com/62705947/199995244-fec56bf8-51c8-4fdf-8cfc-12e59aebf606.png)

Reply:
Resolved